com.skype.Client.appdata.xml

```diff
@@ -25,6 +25,13 @@
   <icon type="remote" height="64" width="64">https://lh3.googleusercontent.com/QfAEt_ya6-n8w_TD9-PsghFC2DMSO7fLGNZB4cQ3RtbBbHFkXJE_gxOc3l32-j6LXg=w64</icon>
   <icon type="remote" height="128" width="128">https://lh3.googleusercontent.com/QfAEt_ya6-n8w_TD9-PsghFC2DMSO7fLGNZB4cQ3RtbBbHFkXJE_gxOc3l32-j6LXg=w128</icon>
   <releases>
+    <release date="2019-05-16" version="8.45.0.41">
+      <description>
+        <ul>
+          <li>null </li>
```

Comment:
```suggestion
          <li>Skype for Web appearance settings for theme and color are now saved after you sign out.</li>
          <li>Quick sign out: When you're using Skype for Web, you can now quickly sign out right from the footer.</li>
          <li>Improvements to easily enable notifications for calls and messages when using Skype for Web.</li>
```

via https://support.skype.com/en/faq/FA34778/release-notes-for-skype-for-windows-mac-linux-and-web

Reply:
I can't see the changelog of 8.45.0.41 in my browser indeed, I can't believe it !!